pmm/pmm2-server-autobuild.groovy

```diff
@@ -106,13 +106,18 @@ pipeline {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                     sh '''
                         sg docker -c "
+                            set -o errexit
+                            set -o xtrace
+
                             export PATH=$PATH:$(pwd -P)/build/bin
 
                             # 1st-party
                             build-server-rpm percona-dashboards grafana-dashboards
-                            build-server-rpm pmm-manage
+                            ### missing rpm spec in PMM-2.0
+                            # build-server-rpm pmm-manage
                             build-server-rpm pmm-managed
-                            build-server-rpm percona-qan-api qan-api
+                            ### missing rpm spec in PMM-2.0
+                            # build-server-rpm percona-qan-api qan-api
```

Comment:
why it is missed?

Reply:
Because we don't have spec for **qan-api**, only for **qan-api2** and **qan-app**. I think it's leftover/copy-paste from PMM1 which was ignored due to lack of `errexit` in pipeline